sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -512,6 +513,10 @@ protected ContextManager getContextManager(Dsn dsn) {
             }
         }
 
+        for (String inAppPackage : inAppPackages) {
+            FrameCache.addAppPackage(inAppPackage);
+        }
```

Comment:
This implicit behavior updating global state seems a little weird but I haven't put a lot of thought into it — I'd expect it doesn't really cause any issues in practice but it does smell a little weird